tools/update-tester/src/main.rs

```diff
@@ -225,25 +278,57 @@ fn main() {
                 password: validate_test_object_matches.value_of("PASSWORD"),
                 database: validate_test_object_matches.value_of("DB"),
             };
+            let mut num_errors = 0;
+            let stdout = io::stdout();
+            let mut out = stdout.lock();
+            let on_error = |test: parser::Test, error: testrunner::TestError| {
+                if num_errors == 0 {
+                    let _ = writeln!(&mut out, "{}\n", "Tests Failed".bold().red());
```

Comment:
I get this after reread but it's a tad confusing and I don't see the value of this header.  Delete?